lib/nmatrix/math.rb

```diff
@@ -112,6 +112,58 @@ def invert
   end
   alias :inverse :invert
 
+  # call-seq:
+  #     invert_exact! -> NMatrix
+  #
+  # Calulates inverse_exact of a matrix of size 2 or 3.
+  # Only works on dense matrices.
+  #
+  # * *Raises* :
+  #   - +DataTypeError+ -> cannot invert an integer matrix in-place.
+  #   - +NotImplementedError+ -> cannot find exact inverse of matrix with size greater than 3  #
+  def invert_exact!
```

Comment:
@mohawkjohn I have made the changes you asked for.
and thank you for letting me know about those command. i will keep that in mind .